pkg/repository/notificationrepository/notification_db_models.go

```diff
@@ -73,9 +73,11 @@ func (n *notificationRow) ToNotificationModel() (models.Notification, error) {
 		// CustomFields is set below
 	}
 
-	err := json.Unmarshal(n.CustomFields, &notification.CustomFields)
-	if err != nil {
-		return empty, err
+	if n.CustomFields != nil {
```

Comment:
This makes sure we handle and empty slice the same way as a nil slice.

```suggestion
	if len(n.CustomFields) > 0 {
```

Reply:
Thanks. 
Suggestion added.